build.gradle.kts

```diff
@@ -25,6 +27,7 @@ buildscript {
     dependencies {
         classpath(PluginDependencies.android)
         classpath(PluginDependencies.kotlin)
+        classpath("com.novoda:bintray-release:${Versions.novodaBintrayRelease}")
```

Comment:
There are problems with this plugin and upgrading the version of Gradle, we are best avoiding it

Reply:
Yes but I think this is a bigger task that we shouldn't handle in this PR.